features/unbeatable_computer.feature

```diff
@@ -0,0 +1,18 @@
+Feature: ubeatable computer
```

Comment:
typo: u`n`beatable

Reply:
resolved in https://github.com/olgaboiar/ttt_python/pull/7/commits/6747559498cb0b457bf25097ede60dc6481edc1a